alacritty_config_derive/src/de_struct.rs

```diff
@@ -135,14 +135,14 @@ fn field_deserializer(field_streams: &mut FieldStreams, field: &Field) -> Result
                     config.#ident = serde::Deserialize::deserialize(unused).unwrap_or_default();
                 });
             },
-            "deprecated" => {
-                // Construct deprecation message and append optional attribute override.
-                let mut message = format!("Config warning: {} is deprecated", literal);
+            "deprecated" | "removed" => {
+                // Construct deprecation/removal message and append optional attribute override.
+                let mut message = format!("Config warning: {} is {}", literal, parsed.ident);
```

Comment:
I don't think "XYZ is removed" is correct. While "is" might be accurate for deprecation, the past tense would need to be used for removal.

Reply:
It's valid English; "removed" can be a past participle just like "deprecated", but I agree it's awkward and not common usage... How about "has been removed" (present perfect tense)?

Reply:
No, the point was that the format should have `was`, but for deprecation it should have `is`.

So the main issue is that `format!("Config warning: {} was {}..."`, but for deprecated we need `format!` with `is`.

Reply:
It both was removed and still is removed. ;) I can change to "was".

Reply:
Now using "was" here.